.github/issue-template.md

```diff
@@ -14,3 +14,16 @@ about: Template for both bug reports and feature requests
 3.
 
 # Additional Info
+
+- Kubernetes version:
+
+  **Output of `kubectl version`:**
+
+  ```
+  (paste your output here)
+  ```
+
+- Tekton version:
```

Comment:
Worth including a helper snippet like

```
kubectl get pods -n tekton-pipelines -l app=tekton-pipelines-controller -o=jsonpath='{.items[0].metadata.labels.version}'
```

?

Reply:
or `tkn version`? I guess this will only work on official releases and on later versions of Tekton and tkn

Reply:
I guess it should say "Tekton pipeline version"

Reply:
Or both :angel: 
I'll update :wink: